docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - GAS_LIMIT=6700000
       - IPFS_HOST=ipfs
       - IPFS_PORT=5001
+      - IPNS_PATH=https://ipfs.io/ipns/
```

Comment:
Could we locally use the local ipfs? Shouldn't our local ipfs support ipns as well? Also, our tests seem to be using the ipns, probably good that our tests don't make calls to https://ipfs.io/ipns